### PR TITLE
(GH-24) Adds a target to .netstandard 2.0

### LINKED ISSUE #24: Target .NET Standard 2.0
Target .NET Standard 2.0

### PR DESCRIPTION
**Pull request by theramis**
Also updated current support from .net46 to .net461

Fixes #24

### CODE REVIEW

**Comment by theramis** [comment]
Not sure why the build is failing. 
spongebob-squarepants-sad-OPU6wzx8JrHna

**Comment by gep13** [comment]
@theramis does the build work locally for you?

**Comment by gep13** [comment]
Can I get you to try adding this:

https://github.com/cake-contrib/Cake.Gem/blob/develop/.appveyor.yml#L1-L4

To the appveyor.yml file?

**Comment by theramis** [comment]
Replicated the issue locally, looking into it now

File: src/Cake.Gulp/Cake.Gulp.csproj [hunk 1]
```diff
@@ -10,22 +10,21 @@
...
+    <DocumentationFile>bin\Release\$(TargetFramework)\Cake.Gulp.xml</DocumentationFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <Compile Include="..\SolutionInfo.cs" Link="SolutionInfo.cs" />
-  </ItemGroup>
-
   <ItemGroup>
-    <PackageReference Include="Cake.Core" Version="0.28.0">
+    <PackageReference Include="Cake.Core" Version="0.30.0">
```
Comment (pascalberger) [inline src/Cake.Gulp/Cake.Gulp.csproj]: As this addin doesn't require anything from 0.28+ there's no need to bump this version and it should be compile against 0.28.0 to provide maximum compatibility.

File: src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <RuntimeFrameworkVersion>2.0.5</RuntimeFrameworkVersion>
     <IsPackable>false</IsPackable>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="..\SolutionInfo.cs" Link="SolutionInfo.cs" />
+    <PackageReference Include="Cake.Core" Version="0.30.0" />
+    <PackageReference Include="Cake.Testing" Version="0.30.0" />
```
Comment (pascalberger) [inline src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj]: As this addin doesn't require anything from 0.28+ there's no need to bump this version and it should be compile against 0.28.0 to provide maximum compatibility.

File: src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
+    <PackageReference Include="Shouldly" Version="3.0.1" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
```
Comment (pascalberger) [inline src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj]: Should be on the same version as XUnit (2.4.0)

File: src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj [hunk 4]
```diff
@@ -1,26 +1,27 @@
...
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
```
Comment (pascalberger) [inline src/Cake.Gulp.Tests/Cake.Gulp.Tests.csproj]: Should be on the same version as XUnit (2.4.0)

**Review by pascalberger — CHANGES_REQUESTED** [review]
@theramis Thanks for your contribution 👍 

I've added some comments for some minor fixes which should be done before merging. Are you in a position to make these?

**Comment by theramis** [comment]
Done @pascalberger 
😄

**Comment by theramis** [comment]
On second thought, after reading this https://xunit.github.io/releases/2.4. Turns out `dotnet-xunit` is not stable so decided to remove it.

**Comment by pascalberger** [comment]
@theramis Thanks 👍 Your changes have been merged and I'll push a new release containing these changes soon

**Pull request merged**